src/services/edusharing/logic/connector.js

```diff
@@ -128,9 +129,15 @@ class EduSharingConnector {
 	async login() {
 		this.authorization = await this.getCookie();
 		this.accessToken = await this.getAuth();
+		this.accessTokenCreationTime = new Date().getTime();
 	}
 
 	isLoggedin() {
+		const TwoHoursAgo = new Date().getTime() - 1000*60*120;
```

Comment:
Codacy found an issue: [Operator '*' must be spaced.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=6005161)